price-feeder/oracle/oracle.go

```diff
@@ -147,16 +147,24 @@ func (o *Oracle) SetPrices() error {
 	g := new(errgroup.Group)
 	mtx := new(sync.Mutex)
 	providerPrices := make(map[string]map[string]provider.TickerPrice)
+	requiredRates := make(map[string]bool)
```

Comment:
just a nifty FYI, typically when you don't care about the value and you wanna just use the map as a "set" data structure, you can instead use `struct{}` as the value type. This has the benefit of requiring no additional space overhead 👍